npm/translatePolicy.go

```diff
@@ -48,7 +49,7 @@ func craftPartialIptablesCommentFromPort(portRule networkingv1.NetworkPolicyPort
 	}
 
 	if portRule.Port != nil {
-		partialComment += "PORT-"
+		partialComment += "PORT_"
```

Comment:
What is this change?.

Reply:
To update comments in Iptable rules like below

"ALLOW-app:backend-IN-ns-testnamespace-AND-PORT_8000-TO-app:frontend-IN-ns-testnamespace"
before:
"ALLOW-app:backend-IN-ns-testnamespace-AND-PORT-8000-TO-app:frontend-IN-ns-testnamespace"